ui/src/components/autocomplete/QasAutocomplete.stories.js

```diff
@@ -0,0 +1,117 @@
+import QasAutocomplete from './QasAutocomplete.vue'
+
+const noSummary = {
+  type: { summary: null }
+}
+
+const options = `{
+  distance: 100,
+  includeScore: true,
+  keys: ['label', 'value'],
+  location: 0,
+  maxPatternLength: 32,
+  minMatchCharLength: 1,
+  shouldSort: true,
+  threshold: 0.1,
+  tokenize: true,
+}`
+
+export default {
+  component: QasAutocomplete,
+  title: 'Components/Autocomplete',
+
+  parameters: {
+    docs: {
+      description: {
+        component: 'Component for select text search that implements [Fuse](https://fusejs.io/) and [QSelect](https://quasar.dev/vue-components/select#Introduction) (all QSelect events are available).'
+      }
+    }
+  },
+
+  argTypes: {
+    labelKey: {
+      description: 'Object key to be replaced by label if there is no key label on any object.'
+    },
+
+    valueKey: {
+      description: 'Object key to be replaced with value if there is no value key in the object.'
+    },
+
+    options: {
+      description: 'Select options.'
+    },
+
+    fuseOptions: {
+      description: 'Library options [Fuse.js](https://fusejs.io/).',
+      table: {
+        defaultValue: {
+          detail: options
+        }
+      }
+    },
+
+    value: {
+      description: 'Value receives the value of the Input and can be multiple.'
+    },
+
+    // events
+    input: {
+      description: 'Raises the input event by passing the value parameter.'
```

Comment:
`description: 'Fires when model changes. Is also used by `v-model`.'`

Reply:
feito